radixconfig.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       variables:
         CLUSTER_EGRESS_IPS: undefined # Comma separated list of IPs of all Public IP Prefixes assigned to the ClusterType (production, playground, development)
     - name: auth
-      image: quay.io/oauth2-proxy/oauth2-proxy:v7.1.3 # see https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/docs/configuration/overview.md
+      image: quay.io/oauth2-proxy/oauth2-proxy:v7.2.0 # see https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/docs/configuration/overview.md
```

Comment:
I believe @nilsgstrabo is looking into updating the oauth2-proxy image in a separate ticket.
I don't think it should be done here.